index.ts

```diff
@@ -99,6 +99,31 @@ async function main() {
             .sort()
             .toString();
           const projectIssueLabelsStringified = getDevpoolIssueLabels(projectIssue, projectUrl).sort().toString();
+
+          // if bounty is unavailable and exists in the list with the state open then close such bounty
+          if (isDevpoolUnavailableLabel && devpoolIssue.state === "open") {
+            await octokit.rest.issues.update({
+              owner: DEVPOOL_OWNER_NAME,
+              repo: DEVPOOL_REPO_NAME,
+              issue_number: devpoolIssue.number,
+              state: "closed",
+            });
+            console.log(`Closed (bounty is unavailable): ${devpoolIssue.html_url} (${projectIssue.html_url})`);
+            continue;
+          }
+
+          // if bounty is available exists in the list with the state closed then reopen such bounty
+          if (!isDevpoolUnavailableLabel && devpoolIssue.state === "closed") {
```

Comment:
Again, no need to check the `Unavailable` label

If project (i.e. partner) issue is NOT assigned to anybody and devpool issue from https://github.com/ubiquity/devpool-directory/issues is "closed" then open such issue